src/ui/api/systeminfo.go

```diff
@@ -86,18 +86,19 @@ var (
 
 //GeneralInfo wraps common systeminfo for anonymous request
 type GeneralInfo struct {
-	WithNotary              bool                             `json:"with_notary"`
-	WithClair               bool                             `json:"with_clair"`
-	WithAdmiral             bool                             `json:"with_admiral"`
-	AdmiralEndpoint         string                           `json:"admiral_endpoint"`
-	AuthMode                string                           `json:"auth_mode"`
-	RegistryURL             string                           `json:"registry_url"`
-	ProjectCreationRestrict string                           `json:"project_creation_restriction"`
-	SelfRegistration        bool                             `json:"self_registration"`
-	HasCARoot               bool                             `json:"has_ca_root"`
-	HarborVersion           string                           `json:"harbor_version"`
-	NextScanAll             int64                            `json:"next_scan_all"`
-	ClairVulnStatus         *models.ClairVulnerabilityStatus `json:"clair_vulnerability_status,omitempty"`
+	WithNotary                  bool                             `json:"with_notary"`
+	WithClair                   bool                             `json:"with_clair"`
+	WithAdmiral                 bool                             `json:"with_admiral"`
+	AdmiralEndpoint             string                           `json:"admiral_endpoint"`
+	AuthMode                    string                           `json:"auth_mode"`
+	RegistryURL                 string                           `json:"registry_url"`
+	ProjectCreationRestrict     string                           `json:"project_creation_restriction"`
+	SelfRegistration            bool                             `json:"self_registration"`
+	HasCARoot                   bool                             `json:"has_ca_root"`
+	HarborVersion               string                           `json:"harbor_version"`
+	NextScanAll                 int64                            `json:"next_scan_all"`
+	ClairVulnStatus             *models.ClairVulnerabilityStatus `json:"clair_vulnerability_status,omitempty"`
+	RegistryStorageProviderName string                           `json:"registry_storage_provider_name"`
```

Comment:
Do we need to consider migration case? Because in previous version, there is no registry_storage_provider_name in harbor.cfg.

Reply:
No migration works for this release.